src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -541,7 +541,7 @@ export default {
           filingType,
           title: this.filingTypeToName(filingType, agmYear),
           filingId: header.filingId,
-          filingAuthor: header.certifiedBy,
+          filingAuthor: header.isStaff ? 'Registry Staff' : (header.certifiedBy || 'Unknown'),
```

Comment:
I wasn't sure if this was worth a new function to call from several places... so I didn't. Thoughts?

Reply:
Oh ya know i'm fine with ternary!

Reply:
I'm ok with that too.